### DISCUSSION TITLE
2021-04-24

### DISCUSSION THREAD

**Discussion by rtsisyk** [organicmaps/organicmaps#334] [Announcements]
(!) Disclaimer. This version changes internal formats and contains breaking changes. If you use bookmarks, please export them via "Export file" before upgrading.

- Rebrand to Organic Maps.
- Remove All Files Access permissions.
- Remove Wi-Fi permissions.
- Remove annoying onboarding screens.
- Improve battery life.
- Add new map data from April 20.
- Remove broken booking, taxi, guides and reviews.
- Enable automatic bookmark backups for Google phones.
- Reworking sharing dialogs.
- Enable full Android 11 support.
- Fix multiple bugs & crashes.

<hr /><em>This discussion was created from the release <a href='https://github.com/organicmaps/organicmaps/releases/tag/2021-04-24'>2021-04-24</a>.</em>

**Comment by keunes** [comment]
Nice! Great job everyone working on this fork 👏
Just out of curiousity: why the change from 'open' to 'organic'? (Is there a discussion/issue I can read up on?)

**Reply by biodranik** [reply]
In short: it's unique, we're writing a *clean* code (without ~~pesticides~~ *trackers*) and we love that name :)
